code-challenges.test.js

```diff
@@ -20,34 +20,72 @@ var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+describe("shuffle", () => {
+    it("shifts and shuffles an array", () => {
+        expect(shuffle(colors1)).toContain("blue", "green", "yellow", "pink");
+        expect(shuffle(colors2)).toContain("indigo", "periwinkle", "ochre", "aquamarine", "saffron");
+    });
 
+    // b) Create the function that makes the test pass.
 
-// b) Create the function that makes the test pass.
+    ///PSEUDO///
 
+    // Create a function that takes in an array, removes the first item from the array and shuffles the remaining content
+    //  make a function that passes an array and removes the first item.
+    // choose a method that will shuffle the remaining items in the array
+    // 
 
+    const shuffle = (array) => {
+        array.shift()
+        for (let i = array.length - 1; i > 0; i--) {
+            let randomArray = Math.random();
+        }
+        return array
+    }
```

Comment:
The `.shift()` looks good. I am not sure if you are changing the order of your array with this code.